conf.d/_tide_init.fish

```diff
@@ -1,17 +1,28 @@
+# Gets the parent of the parent of a given path
+# Returns the current directory if the given path does not have two parents
+function _tide_dirname_2 -a somepath
```

Comment:
Do we really need to call this `_tide_dirname_2`? It seems to me that just `_tide_dirname` would suffice.